src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php

```diff
@@ -99,7 +99,7 @@ public function it_allows_updating_a_subscription(): void
             'sylius_resource[email]' => 'biff.tannen@bttf.com',
         ]);
 
-        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
+        //$this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
```

Comment:
Is it possible to international?

Reply:
That was a mistake, the redirection is OK.